src/NUnitEngine/nunit.engine.tests/Services/TestRunnerFactoryTests/Results/Net20SingleProjectExpectedRunnerResults.cs

```diff
@@ -0,0 +1,236 @@
+            }
+
+            throw new ArgumentOutOfRangeException(
+                $"No expected Test result provided for this {nameof(ProcessModel)}/{nameof(DomainUsage)} combination.");
+        }
+
+        private static RunnerResult GetProcessModelMultipleResult(DomainUsage domainUsage)
+        {
+            switch (domainUsage)
+            {
+                case DomainUsage.Default:
+                    return new RunnerResult
+                    {
+                        TestRunner = typeof(MultipleTestProcessRunner),
+                        SubRunners = new[]
+                        {
+                            new RunnerResult {TestRunner = typeof(ProcessRunner)}
+                        }
+                    };
+                case DomainUsage.None:
+                    return new RunnerResult
+                    {
+                        TestRunner = typeof(MultipleTestProcessRunner),
+                        SubRunners = new[]
+                        {
+                            new RunnerResult {TestRunner = typeof(ProcessRunner)}
+                        }
+                    };
+                case DomainUsage.Single:
+                    return new RunnerResult
+                    {
+                        TestRunner = typeof(MultipleTestProcessRunner),
+                        SubRunners = new[]
+                        {
+                            new RunnerResult {TestRunner = typeof(ProcessRunner)}
+                        }
+                    };
+                case DomainUsage.Multiple:
+                    return new RunnerResult
+                    {
+                        TestRunner = typeof(MultipleTestProcessRunner),
+                        SubRunners = new[]
+                        {
+                            new RunnerResult {TestRunner = typeof(ProcessRunner)}
+                        }
+                    };
+                default:
+                    ThrowOutOfRange(domainUsage);
+                    break;
+            }
+            return null;
+        }
+
+        private static RunnerResult GetProcessModelSeparateResult(DomainUsage domainUsage)
+        {
+            switch (domainUsage)
+            {
+                case DomainUsage.Default:
+                    return new RunnerResult
+                    {
+                        TestRunner = typeof(ProcessRunner)
+                    };
+                case DomainUsage.None:
+                    return new RunnerResult
+                    {
+                        TestRunner = typeof(ProcessRunner)
+                    };
+                case DomainUsage.Single:
+                    return new RunnerResult
+                    {
+                        TestRunner = typeof(ProcessRunner)
+                    };
+                case DomainUsage.Multiple:
+                    return new RunnerResult
+                    {
+                        TestRunner = typeof(ProcessRunner)
+                    };
```

Comment:
Maybe stack the cases like this?

```cs
                case DomainUsage.Default:
                case DomainUsage.None:
                case DomainUsage.Single:
                case DomainUsage.Multiple:
                    return new RunnerResult
                    {
                        TestRunner = typeof(ProcessRunner)
                    };
```

Or if this was the intent, maybe this option?

```cs
        private static RunnerResult GetProcessModelSeparateResult(DomainUsage domainUsage)
        {
            if (!Enum.IsDefined(typeof(DomainUsage), domainUsage))
                ThrowOutOfRange(domainUsage);

            return new RunnerResult
            {
                TestRunner = typeof(ProcessRunner)
            };
        }
```

Reply:
I think the first option is good. 🙂 I'd been keeping these tests as explicit as possible, but that would help on the maintenance front!